osu.Game/Screens/OnlinePlay/DrawableRoomPlaylistItem.cs

```diff
@@ -191,12 +203,21 @@ protected override Drawable CreateContent()
                         {
                             new Drawable[]
                             {
+                                ownerAvatar = new OwnerAvatar
```

Comment:
Adding the avatar at this level means that it will also appear in playlists with the room creator as the owner of all playlist items, which feels unintended / somewhat of a waste of space:

![osu_2021-11-25_20-34-47](https://user-images.githubusercontent.com/20418176/143493472-de1a16c4-608a-4c43-b4f3-3455f84fa3e6.jpg)

Are there any plans regarding that going forward, or is that just an oversight?

Reply:
Feels like it definitely shouldn't be present in all but the multiplayer queue context.